app/controllers/admin/routes.py

```diff
@@ -432,16 +438,35 @@ def manageServiceLearningCourses(term=None):
     manageTerm = Term.get_or_none(Term.id == term) or g.current_term
 
     setRedirectTarget(request.full_path)
-
+    # retrieve and store the courseID of the imported course from a session variable if it exists. 
+    # This allows us to export the courseID in the html and use it.
+    courseID = session.get("alterCourseId")  
+    
+    if courseID:
+        # delete courseID from the session if it was retrieved, for storage purposes.
+        session.pop("alterCourseId")  
+        return render_template('/admin/manageServiceLearningFaculty.html',
+                                courseInstructors = getInstructorCourses(),
+                                unapprovedCourses = unapprovedCourses(term),
```

Comment:
I think in your merge conflict you may have overwritten a bug patch we put in. All calls using `term` as a parameter need to use `manageTerm` instead.

Reply:
Here and in the render_template below it

Reply:
Fixed